aeat.py

```diff
@@ -1155,6 +1166,8 @@ def get_received_sii_reports(cls):
                     ('sii_pending_sending', '=', True),
                     ('type', '=', 'in'),
                     ('move', '!=', None),
+                    ('invoice_date', '<=', today - timedelta(days=config.sii_default_limit_date)),
+                    ('accounting_date', '=', None),
```

Comment:
The domain is not correct, shall be (missing apply format with 79 cols):

['OR', [
('invoice_date', '<=', today - timedelta(days=config.sii_default_limit_date if config.sii_default_limit_date else 0)),
('accounting_date', '=', None),
 ], [
('accounting_date', '<=', today - timedelta(days=config.sii_default_limit_date if config.sii_default_limit_date else 0)),
 ]]